app/views/discussions/show.html.erb

```diff
@@ -41,8 +41,10 @@
 
 <% end %>
 
-<div class="discussion-actions">
-  <% @discussion.reachable_statuses_for(current_user).each do |status| %>
-    <%= discussion_update_status_button(status) %>
-  <% end if current_user %>
-</div>
+<% if current_user&.moderator_here? %>
```

Comment:
Although initiators can also change discussion status, we are hiding that button as requested by @rgonzalezt. They will still change it when solving an assignment.